src/neo-vm/Types/StackItem.cs

```diff
@@ -32,37 +28,9 @@ public static StackItem FromInterface<T>(T value)
             return new InteropInterface<T>(value);
         }
 
-        public virtual BigInteger GetBigInteger()
-        {
-            return new BigInteger(GetByteArray());
-        }
-
-        public abstract bool GetBoolean();
-
-        public abstract ReadOnlySpan<byte> GetByteArray();
-
-        public virtual int GetByteLength()
-        {
-            return GetByteArray().Length;
-        }
-
-        public override int GetHashCode()
-        {
-            unchecked
-            {
-                int hash = 17;
-                foreach (byte element in GetByteArray())
-                    hash = hash * 31 + element;
-                return hash;
-            }
-        }
-
-        public virtual string GetString()
-        {
-            return Encoding.UTF8.GetString(GetByteArray());
-        }
+        public abstract override int GetHashCode();
```

Comment:
since you overrode equals in this class, you need to provide a matching implementation of GetHashCode